docs/usage_and_configuration/ways_to_install_black.md

```diff
@@ -0,0 +1,46 @@
+# Ways to install Black
+
+There are several ways you can install _Black_.
+
+## Install from PyPI
+
+To install the latest release of _Black_ from [PyPI] (Requires Python 3.8+), use:
```

Comment:
I wonder if we can link to the pyproject.toml directive here to avoid having _yet_ another place to upgrade when we move to requiring >= 3.9 - All for ideas here but can live with it if it's to hard ...

Reply:
I'm checking if this can be done via tomli and Sphinx configuration somehow.